.github/workflows/canary.yml

```diff
@@ -25,27 +25,27 @@ jobs:
           FORCE_COLOR: 0
         with:
           node-version: '18'
-          cache: 'yarn'
+          cache: 'npm'
           registry-url: 'https://registry.npmjs.org'
 
       - name: Install Dependencies
-        run: yarn --frozen-lockfile
+        run: npm ci
```

Comment:
```suggestion
        run: npm ci --prefer-offline
```

Reply:
And elsewhere

Reply:
what is that for? just did a quick google and wouldn't that also require to specify the cache location? 🤔

I think it's best to keep `npm ci` for now and then in a separate PR check the difference using `--prefer-offline` as I'm curious how big of a difference it would be 🤗

Reply:
Speeds up ci a bit. Not critical